Apps/Sandcastle/gallery/Reverse Geocoder.html

```diff
@@ -0,0 +1,175 @@
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+    require.config({
+        baseUrl : '../../../Source',
+        waitSeconds : 60
+    });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+    #toolbar {
+        background: rgba(42, 42, 42, 0.8);
+        padding: 4px;
+        border-radius: 4px;
+    }
+    #toolbar input {
+        vertical-align: middle;
+        padding-top: 2px;
+        padding-bottom: 2px;
+    }
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+/**
+ * This class is an example of reverse geocoder using the default geocoder service (Bing Maps)
+ * It also includes click handling and setting points at found or clicked locations.
+ * To reverse geocode, click on any location on the globe to show the address.
+ */
+
+/**
+* Object for both input and result. Should be in external file, to avoid code redundancy.
+*/
+var geoLocation = {
+         latitude: 0.00,
+         longitude: 0.00,
+         address: 'default address'
+};
+
+var viewer = new Cesium.Viewer('cesiumContainer');
+
+var baseUrl = 'https://dev.virtualearth.net/REST/v1/Locations/';
+
+/**
+ * The function called to reverse geocode using this geocoder service.
+ *
+ * @param {String} input The query to be sent to the geocoder service
+ * Calling loadJsonp to bypass CORS on BingMaps
+ */
+    function getAddressFromLocation(geoLocation) {
+       var url = baseUrl + geoLocation.latitude + "," + geoLocation.longitude;
+
+       var promise = Cesium.loadJsonp(url, {
+          parameters : {
+             key : Cesium.BingMapsApi.getKey()
+         },
+         callbackParameterName : 'jsonp'
+       });
+
+       promise.then(function(result) {
+          geoLocation.address = result.resourceSets[0].resources[0].name;
+          setAddressPoint();
+       }).otherwise(function(error) {
+           alert("ERROR getting address: " + error);
+       });
+       // We are not zooming in on the location.
+  }
+/*********************************************************************
+ * Functions below, could be in an external file for this and Custom Geocoder.
+ * geoLocation object as well.
+ ********************************************************************/
+/**
+ * Listen for the geocode search to be complete, to mark it with a point.
+ */
+ viewer.geocoder.viewModel.complete.addEventListener(function () {
+     if (!viewer.geocoder.viewModel.searchText.empty){
+         setPointForSearchLocation();
+     }
+ });
+
+ /**
+ * Find the camera position after search, and put a point there.
+ */
+  function setPointForSearchLocation() {
```

Comment:
Something in this function is causing an error to be thrown when you use the geocoder search